### PR TITLE
add deployment script for adding bAssets to basket manager

### PR DESCRIPTION
**Pull request by cwsnt**
Adding deployment script for adding bAssets to basket manager

### CODE REVIEW

File: deploy/7_AddBassetInBasketManager.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { constants } from "ethers";
+import { DeployFunction } from "hardhat-deploy/types";
+
+const func: DeployFunction = async ({
+  deployments,
+  getNamedAccounts,
+}) => {
+  const network = deployments.getNetworkName();
+  const {deployer} = await getNamedAccounts();
+  let bAssets = [
```
Comment (tjcloa) [inline deploy/7_AddBassetInBasketManager.ts:10]: pls set bAssets by network in one if...else block.

File: deploy/7_AddBassetInBasketManager.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "0x6b41566353d6c7b8c2a7931d498f11489dacac29", // ZUSD Testnet
+    "0xcb46c0ddc60d18efeb0e586c17af6ea36452dae0", // DOC Tesnet
+  ];
+  let factors = [1,1];
+  let bridges = [constants.AddressZero, constants.AddressZero];
+  let mins = [10,10];
+  let maxs = [100,100];
```
Comment (tjcloa) [inline deploy/7_AddBassetInBasketManager.ts:17]: maxs and mins should be null. pls verify if it works correctly. there should not be any restrictions except 1:1 exchange.

**Review by tjcloa — CHANGES_REQUESTED** [review]
pls merge from the origin (target) branch and address inline comments

File: deploy/7_AddBassetInBasketManager.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  let pausedFlags = [false, false];
+
+  if(network === 'rskMainnet') {
+    bAssets = [
+      "0xdb107fa69e33f05180a4c2ce9c2e7cb481645c2d", // ZUSD Mainnet
+      "0xe700691da7b9851f2f35f8b8182c69c53ccad9db" // DOC Mainnet
+    ];
+  }
+
+  console.log("adding bAssets...")
```
[the remark this answers is not in the document]
Comment (cwsnt) [reply deploy/7_AddBassetInBasketManager.ts]: updated in this [commit](https://github.com/DistributedCollective/mynt/pull/5/commits/b791ecb597f6586e57a2ee71466f6b0984cb8889)

File: deploy/7_AddBassetInBasketManager.ts (hunk 1 above)
Comment (cwsnt) [reply deploy/7_AddBassetInBasketManager.ts]: updated in this [commit](https://github.com/DistributedCollective/mynt/pull/5/commits/b791ecb597f6586e57a2ee71466f6b0984cb8889)

File: deploy/7_AddBassetInBasketManager.ts (hunk 2 above)
Comment (cwsnt) [reply deploy/7_AddBassetInBasketManager.ts]: updated in this [commit](https://github.com/DistributedCollective/mynt/pull/5/commits/b791ecb597f6586e57a2ee71466f6b0984cb8889)

**Review by tjcloa — APPROVED** [review]

**Pull request merged**